### DISCUSSION TITLE
scope access tokens to packages

### DISCUSSION THREAD

**Discussion by luwol03** [npm/feedback#595] [Registry] [answered]
As the title said. It would be great if we can scope tokens to specific actions/packages. I dont like it If I pass a token as a secret to a CI system which has full control over my packages. This token should only be valid to push specific packages used in that project.

**Comment by MylesBorins** [comment, marked as answer]
@luwol03 we have plans and are scoping a solution for package scoped tokens. Unfortunately it is not something that we will be bringing to the market. It is being tracked in [this roadmap issue](https://github.com/npm/roadmap/issues/10)
